### PR TITLE
[config/show] Add CLI support for proxy arp

### PR DESCRIPTION
**Pull request by theasianpianist**
Signed-off-by: Lawrence Lee <lawlee@microsoft.com>

**- What I did**
Enable CLI management for proxy ARP setting
**- How I did it**
* Add `config vlan proxy_arp <vid> <enable/disable>
    * Edits VLAN_INTERFACE table to set the "proxy_arp" setting for an interface
    * Restarts the `ndppd` daemon running inside the SWSS container to grab changes
* Add proxy ARP info to `show vlan brief`

**- How to verify it**
* Run unit tests

**- Previous command output (if the output of a command-line utility has changed)**
```
> show vlan brief
+-----------+-----------------+------------+----------------+-----------------------+
|   VLAN ID | IP Address      | Ports      | Port Tagging   | DHCP Helper Address   |
+===========+=================+============+================+=======================+
|      1000 | 192.168.0.1/21  | Ethernet4  | untagged       | 192.0.0.1             |
|           | fc02:1000::1/64 | Ethernet8  | untagged       | 192.0.0.2             |
|           |                 | Ethernet12 | untagged       | 192.0.0.3             |
|           |                 | Ethernet16 | untagged       | 192.0.0.4             |
+-----------+-----------------+------------+----------------+-----------------------+
|      2000 | 192.168.0.10/21 | Ethernet24 | untagged       | 192.0.0.1             |
|           | fc02:1011::1/64 | Ethernet28 | untagged       | 192.0.0.2             |
|           |                 |            |                | 192.0.0.3             |
|           |                 |            |                | 192.0.0.4             |
+-----------+-----------------+------------+----------------+-----------------------+
```

**- New command output (if the output of a command-line utility has changed)**
```
> show vlan brief
+-----------+-----------------+------------+----------------+-----------------------+-------------+
|   VLAN ID | IP Address      | Ports      | Port Tagging   | DHCP Helper Address   | Proxy ARP   |
+===========+=================+============+================+=======================+=============+
|      1000 | 192.168.0.1/21  | Ethernet4  | untagged       | 192.0.0.1             | disabled    |
|           | fc02:1000::1/64 | Ethernet8  | untagged       | 192.0.0.2             |             |
|           |                 | Ethernet12 | untagged       | 192.0.0.3             |             |
|           |                 | Ethernet16 | untagged       | 192.0.0.4             |             |
+-----------+-----------------+------------+----------------+-----------------------+-------------+
|      2000 | 192.168.0.10/21 | Ethernet24 | untagged       | 192.0.0.1             | enabled     |
|           | fc02:1011::1/64 | Ethernet28 | untagged       | 192.0.0.2             |             |
|           |                 |            |                | 192.0.0.3             |             |
|           |                 |            |                | 192.0.0.4             |             |
+-----------+-----------------+------------+----------------+-----------------------+-------------+
```

### CODE REVIEW

**Comment by theasianpianist** [comment]
retest default please

**Comment by theasianpianist** [comment]
retest this please

**Comment by theasianpianist** [comment]
retest this please

**Comment by theasianpianist** [comment]
retest this please

File: config/vlan.py [hunk 1]
```diff
@@ -50,6 +51,34 @@ def del_vlan(db, vid):
         db.cfgdb.set_entry('VLAN_MEMBER', k, None)
     db.cfgdb.set_entry('VLAN', 'Vlan{}'.format(vid), None)
 
+def restart_ndppd():
+    docker_exec_cmd = "docker exec -it swss {}"
```
Comment (prsunny) [inline config/vlan.py:55]: Can you check the behavior when swss docker is not running and user configures proxy_arp?

File: config/vlan.py [hunk 2]
```diff
@@ -50,6 +51,34 @@ def del_vlan(db, vid):
...
+    log.log_info("'setting proxy ARP to {} for Vlan{}".format(mode, vid))
+
+    ctx = click.get_current_context()
+
+    vlan = 'Vlan{}'.format(vid)
+
+    if not clicommon.is_valid_vlan_interface(db.cfgdb, vlan):
+        ctx.fail("Interface {} does not exist".format(vlan))
+
+    db.cfgdb.set_entry('VLAN_INTERFACE', vlan, {"proxy_arp": mode})
```
Comment (prsunny) [inline config/vlan.py:80]: Please check if `mode == enabled/disabled`, if not print a message on the accepted values

File: config/vlan.py (hunk 1 above)
Comment (theasianpianist) [reply config/vlan.py:55]: It should just be something like this:
```
Error response from daemon: Container a183b127127877c76698526e4283491f1fb45243d2e9a3ab80368214f597ff25 is not running
```

Is that fine, or should I check if the container is up first?

File: config/vlan.py (hunk 2 above)
Comment (theasianpianist) [reply config/vlan.py:80]: Click already checks if it is either enabled/disabled, see line 66

File: config/vlan.py (hunk 1 above)
Comment (prsunny) [reply config/vlan.py:55]: I think yes, if user configures proxy-arp and see this error, it might be confusing whether it got applied.

**Review by prsunny — APPROVED** [review]

**Comment by theasianpianist** [comment]
retest this please

**Pull request merged**